product-payment.html

```diff
@@ -0,0 +1,290 @@
+                  </g>
+                  <defs>
+                    <clipPath id="clip0_111_2510">
+                      <rect width="16" height="16" fill="white" />
+                    </clipPath>
+                  </defs>
+                </svg>
+                <p class="product-page__retreat">Retiro en Tienda</p>
+              </a>
+            </div>
+          </div>
+          <div class="product-page__add">
+            <div class="product-page__quantity-selector">
+              <a href="#" class="product-page__quantity__link"></a>
+              <button class="product-page__decrease"></button>
+              <span class="product-page__quantity">---</span>
```

Comment:
Este elemento es para meter la cantidad. Como esta, no puedo meter la cantidad
```suggestion
              <input type="number" class="product-page__quantity" placeholder="---">
```